pkg/migrate/migrate.go

```diff
@@ -79,6 +106,37 @@ func Cli() {
 
 	output := log.New(os.Stdout, "", 0) // this has no time prefix etc
 
+	// escape hatch - for DEV/TEST ONLY
```

Comment:
if this is for dev/test only, that should be mentioned in the flag description - or the flag should be hidden.

Reply:
I'll update the comment. I added this in the case where there is a rare scenario where someone may a justifiable reason to ignore a PVC access volume validation.